src/objects/domain_obj.f90

```diff
@@ -1016,20 +1016,26 @@ subroutine setup_sleve(this, options)
             ! ( Although an argument could be made to calculate this on the offset (u/v) grid b/c that is most
             !   relevant for advection? In reality this is probably a sufficient approximation, as long as we
             !   aren't pushing the gamma factor too close to zero )
-            allocate(gamma_n(this%kds : this%kde+1))
+            ! allocate(gamma_n(this%kds : this%kde+1))
+            allocate(gamma_n(this%kms : max_level))  ! max_level +1 maybe??
+            ! if (options%parameters%debug .and. this_image()==1) write(*,*) " gamma_n: ", gamma_n
```

Comment:
can you clean up the commented out code?

Reply:
done